compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -3746,4 +3746,4 @@ THRIFT_REGISTER_GENERATOR(go, "Go",
                           "    read_write_private\n"
                           "                     Make read/write methods private, default is public Read/Write\n" \
                           "    use_context\n"
-                          "                     Make service method receive a context as first argument.\n")
+                          "                     Make service method receive a context as first argument, only go1.7+ is supported.\n")
```

Comment:
And its also painful to use build tags in the generated code, i think a control flag is still required to decided whether to use `context` or `x/net/context`.